backend/server/common/server.ts

```diff
@@ -20,6 +21,7 @@ export default class ExpressServer {
 
   constructor() {
     const root = path.normalize(__dirname + '/../..');
+    app.use(cors);
```

Comment:
This will allow CORS requests from all origins to succeed. We should nail it down in the future.

Reply:
See #12.